app/javascript/components/key-performance-indicators/components/total-follow-up-meetings/component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { OptionsBox } from "components/dashboard";
+import DateRangeSelect from "../date-range-select";
+import SingleAggregateMetric from "../single-aggregate-metric";
+
+const TotalFollowupMeetings = () => {
+  const threeMonthsAgo = new Date();
+
+  threeMonthsAgo.setMonth(threeMonthsAgo.getMonth() - 3);
+  const dateRanges = [
+    {
+      value: "3-months",
+      name: "Last 3 Months",
+      from: threeMonthsAgo,
+      to: new Date()
+    }
+  ];
+
+  return (
+    <OptionsBox
+      title="Total Follow-Up Meetings"
```

Comment:
I18n

Reply:
This was an incomplete KPI so I just removed it! Thanks for the catch :)